sepsis-quiz-js/src/sepsisQuiz.js

```diff
@@ -116,11 +119,6 @@ class SepsisQuiz {
       }
     }
 
-    /* true or false */
```

Comment:
No, it didn't cause a bug. But this "functionality" isn't implemented in the rest of the logic of the application so if you saw this code and tried to use it, it wouldn't work. Removing completely is safer.